courses.php

```diff
@@ -1,46 +1,47 @@
 <?php include("top.php"); ?>
 <?php include("nav.php");?>
 <div style="float: right;"/><iframe style="padding: 10px; width:120px;height:240px;" marginwidth="0" marginheight="0" scrolling="no" frameborder="0" src="//ws-na.amazon-adsystem.com/widgets/q?ServiceVersion=20070822&OneJS=1&Operation=GetAdHtml&MarketPlace=US&source=ss&ref=as_ss_li_til&ad_type=product_link&tracking_id=drchu02-20&marketplace=amazon&region=US&placement=1530051126&asins=1530051126&linkId=2ea6c883c6cf11f29568856139bad34b&show_border=true&link_opens_in_new_window=true"></iframe></div>
-        <h2>Courses using Python for Everybody</h2>
+        <h2>Cursos que usan Python Para Todos</h2>
         <p>
-This page reflects various courses that use this textbook around the world.
-This translation / adaptation / reuse is encouraged since the materials are
-avilable under a Creative Commons license.
+
+Esta página muestra varios cursos alrededor del mundo que usan este libro.
+Esta traducción/ adaptación/ reutilización es alentada, ya que los materiales
```

Comment:
Minor suggestion: Keep the space before the slashes. 
From
traducción/ adaptación/ reutilización
To
traducción / adaptación / reutilización

Reply:
listo, efectuada la modificación correspondiente. 

 ¿Cuál es el motivo de ésta?

Reply:
Solo una sugerencia de estilo, para mantener el formato original:

"-This translation / adaptation / reuse is encouraged since the materials are"